taxi/models.py

```diff
@@ -0,0 +1,32 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+import taxi_service.settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=63, unique=True)
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.SET_NULL, null=True

```

Comment:
In case you place some arguments on a separate line, place all arguments on a separate line. Besides that, add a `related_name` parameter for the `ForeignKey` field.